Makefile

```diff
@@ -72,6 +72,12 @@ define DOCKER
 	docker run --rm -e IN_DOCKER=1 --user $(shell id -u):$(shell id -g) -w /go/src/$(REPOPATH) -v $(GOPATH):/go --entrypoint /bin/bash $(1) -c '$(2)'
 endef
 
+# Newline for warnings
+define N
+
+
+endef
```

Comment:
````
	$(warning ******************************************************************************)
	$(warning WARNING: You are building minikube outside the expected GOPATH:)
	$(warning )
	$(warning expected: $(GOPATH)/src/$(REPOPATH) )
	$(warning   got:      $(PWD) )
	$(warning )
	$(warning You will likely encounter unusual build failures. For proper setup, read: )
	$(warning https://github.com/kubernetes/minikube/blob/master/docs/contributors/build_guide.md)
	$(warning ******************************************************************************)
````

produces: 

````

Makefile:108: ******************************************************************************
Makefile:108: WARNING: You are building minikube outside the expected GOPATH:
Makefile:108: 
Makefile:108: expected: ****/go/src/k8s.io/minikube 
Makefile:108: got:      ******/minikube 
Makefile:108: 
Makefile:108: You will likely encounter unusual build failures. For proper setup, read: 
Makefile:108: https://github.com/kubernetes/minikube/blob/master/docs/contributors/build_guide.md
Makefile:108: ******************************************************************************

`````

Reply:
That's a much better soultion. Why didn't I think of that?

(done)